README.md

```diff
@@ -51,20 +51,13 @@ pip install deepchecks #--user
 ```
 
 ### From source
-First clone the repository and then install the package from inside the repository's directory:
+For latest stable version:
 ```bash
-git clone https://github.com/deepchecks/deepchecks.git
-cd deepchecks
-# for installing stable tag version and not the latest commit to main
-git checkout tags/<name_of_latest_tag>
+pip install git+https://github.com/deepchecks/deepchecks.git@0.0.10
```

Comment:
do we want to have the version in the README? we'll have to update on each release

Reply:
I think it's usually like that, I also changed that according to gabbay's comment

Reply:
I don't think it gives value to add which version
Lets leave it as is

Reply:
ok